ros2_batch_job/__main__.py

```diff
@@ -319,14 +320,23 @@ def build_and_test(args, job):
     if ret_build:
         return ret_build
 
+    if args.enable_sanitizer_type:
+        ret_build_with_sanitizer = build_with_sanitizer(cmd, args, job)
+    if ret_build_with_sanitizer:
```

Comment:
This line will fail if sanitizer is not enabled. The condition should be indented one level.

Reply:
Good catch! I have avoided this logic for now